bin/pt-ioprofile

```diff
@@ -203,9 +204,15 @@ parse_options() {
          _parse_config_files "$user_config_file"
       done
    else
-      _parse_config_files "/etc/percona-toolkit/percona-toolkit.conf" "/etc/percona-toolkit/$TOOL.conf"
+       GLOBAL_CONFIG=1
+      _parse_config_files "/etc/percona-toolkit/percona-toolkit.conf"
+       GLOBAL_CONFIG=0
+       _parse_config_files "/etc/percona-toolkit/$TOOL.conf"
       if [ "${HOME:-}" ]; then
-         _parse_config_files "$HOME/.percona-toolkit.conf" "$HOME/.$TOOL.conf"
+         GLOBAL_CONFIG=1
+         _parse_config_files "$HOME/.percona-toolkit.conf"
+         GLOBAL_CONFIG=0
+         _parse_config_files "$HOME/.$TOOL.conf"
```

Comment:
what is the logic firstly set GLOBAL_CONFIG=1 and then GLOBAL_CONFIG=0?
maybe else statement is missed?

Reply:
This change is part of lib/bash/parse_options.sh GLOBAL_CONFIG variable is used to distinguish config file, applied globally to the toolkit and which can contain options, not available for particular tool, for the tool-specific config. Please check code at line 479: https://github.com/percona/percona-toolkit/blob/c119293bae29767b66a733b9be904861933e6d79/lib/bash/parse_options.sh#L478

Reply:
It is possible to rewrite the function to take GLOBAL_CONFIG as a parameter, but this function is called in many places/tools that rely on the old behavior.